src/main/java/nl/knmi/geoweb/backend/product/sigmet/Sigmet.java

```diff
@@ -221,6 +221,7 @@ public Sigmet(Sigmet otherSigmet) {
 		this.issuedate = otherSigmet.getIssuedate();
 		this.firFeature = otherSigmet.firFeature;
 		this.type=otherSigmet.type;
+		this.change= otherSigmet.change;
```

Comment:
I think this should be corrected, for two reasons:
* change is a private member, so it should be otherSigmet.getChange()
* at line 212 the change is already set